### ISSUE TITLE
VM Start SELinux Error

### ISSUE THREAD

**Issue by drag0n141** [kubevirt/kubevirt#11525]
I want to start am VM but its not possible. I get only this message. selinux or apparmor is not installed on the host.

```
{"component":"virt-handler","level":"error","msg":"failed to configure pod network","pos":"netpod.go:170","reason":"could not retrieve pid 245420 selinux label: operation not supported","timestamp":"2024-03-15T14:36:16.630989Z"}
```

**Environment**:
- KubeVirt version (use `virtctl version`): v1.2.0
- Kubernetes version (use `kubectl version`): 1.29.2
- OS (e.g. from /etc/os-release): Debian 12
- Kernel (e.g. `uname -a`): 6.1.0-17

**Comment by andiolsi** [comment]
Hi,  can confirm your finding and think this is a bug.

When selinux is not installed virt-handler logs:
`{"component":"virt-handler","level":"error","msg":"failed to configure pod network","pos":"netpod.go:170","reason":"could not retrieve pid 2112031 selinux label: operation not supported","timestamp":"2024-03-16T12:13:39.177640Z"}
{"component":"virt-handler","kind":"","level":"error","msg":"Synchronizing the VirtualMachineInstance failed.","name":"vm1","namespace":"kubevirt","pos":"vm.go:2002","reason":"failed to configure vmi network: setup failed, err: Critical network error: could not retrieve pid 2112031 selinux label: operation not supported","timestamp":"2024-03-16T12:13:39.177700Z","uid":"ea7792ab-110d-435e-b031-c9e56de284a6"}
{"component":"virt-handler","level":"error","msg":"couldn't find the SELinux context for vm1","pos":"vm.go:1366","reason":"no data available","timestamp":"2024-03-16T12:13:39.185144Z"}
{"component":"virt-handler","level":"error","msg":"virt-launcher crashed due to a network error. Updating VMI vm1 status to Failed","pos":"vm.go:1459","timestamp":"2024-03-16T12:13:39.185181Z"}
{"component":"virt-handler","level":"info","msg":"re-enqueuing VirtualMachineInstance kubevirt/vm1","pos":"vm.go:1673","reason":"failed to configure vmi network: setup failed, err: Critical network error: could not retrieve pid 2112031 selinux label: operation not supported","timestamp":"2024-03-16T12:13:39.190814Z"}`

I proceeded to install the packages  `selinux-basics` and `selinux-utils`. 

For good measure i gave the machine a reboot.

Afterwards the VMs start and there are no error messages logged.


I would tend to think this is unwanted behaviour.

**Comment by sebt3** [comment]
/remove-lifecycle stale

Still completly fresh. On debian. As suggested by [andiolsi](https://github.com/andiolsi) :
```bash
sudo apt install selinux-basics selinux-utils
```
fixed it for me too.
